tf2onnx/graph.py

```diff
@@ -425,6 +425,8 @@ def __init__(self, nodes, output_shapes=None, dtypes=None, target=None, opset=No
         self._nodes = []
         self._nodes_by_name = {}
         self._output_to_node_name = {}
+        self._input_to_node_name = {}
```

Comment:
Personally I would call this _output_to_consumers to match _find_output_consumers and _output_to_node_name.  We've been a little inconsistent with naming, but I think it makes most sense to consider these strings ("node_name:0") to be "node outputs".  We've also referred to them as edges.  @guschmue Any thoughts?

Reply:
I like _output_to_consumers would be good since it tells directly what is stored.

Reply:
Done.